[2025-04-11T08:59:37.724+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_and_verify_data.run_generate_script manual__2025-04-11T08:54:34.942760+00:00 [queued]>
[2025-04-11T08:59:37.740+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_and_verify_data.run_generate_script manual__2025-04-11T08:54:34.942760+00:00 [queued]>
[2025-04-11T08:59:37.742+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2025-04-11T08:59:37.765+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): run_generate_script> on 2025-04-11 08:54:34.942760+00:00
[2025-04-11T08:59:37.772+0000] {standard_task_runner.py:57} INFO - Started process 9670 to run task
[2025-04-11T08:59:37.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_and_verify_data', 'run_generate_script', 'manual__2025-04-11T08:54:34.942760+00:00', '--job-id', '1831', '--raw', '--subdir', 'DAGS_FOLDER/dags_1.py', '--cfg-path', '/tmp/tmph8zzisfn']
[2025-04-11T08:59:37.786+0000] {standard_task_runner.py:85} INFO - Job 1831: Subtask run_generate_script
[2025-04-11T08:59:37.884+0000] {task_command.py:415} INFO - Running <TaskInstance: generate_and_verify_data.run_generate_script manual__2025-04-11T08:54:34.942760+00:00 [running]> on host 0821f6066d6a
[2025-04-11T08:59:38.027+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_and_verify_data' AIRFLOW_CTX_TASK_ID='run_generate_script' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T08:54:34.942760+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T08:54:34.942760+00:00'
[2025-04-11T08:59:38.030+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-11T08:59:38.033+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/data/ && python3 /opt/***/data/generate_script.py && echo "Script executed successfully." || echo "Script failed."']
[2025-04-11T08:59:38.049+0000] {subprocess.py:86} INFO - Output:
[2025-04-11T08:59:39.981+0000] {subprocess.py:93} INFO - Script executed successfully.
[2025-04-11T08:59:39.982+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-11T08:59:40.018+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=generate_and_verify_data, task_id=run_generate_script, execution_date=20250411T085434, start_date=20250411T085937, end_date=20250411T085940
[2025-04-11T08:59:40.054+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-11T08:59:40.122+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
